docs/src/keps/kep-001.rst

```diff
@@ -0,0 +1,172 @@
+
+2. An **Informational** KEP describes a Klio design issue, or provides general guidelines or information to the Python community, but does not propose a new feature. Informational KEPs do not necessarily represent a Klio community consensus or recommendation, so users and implementers are free to ignore Informational KEPs or follow their advice.
+
+3. A **Process** KEP describes a process surrounding Klio, or proposes a change to (or an event in) a process. Process KEPs are like Standards Track KEPs but apply to areas other than the Klio language itself. They may propose an implementation, but not to Klio's codebase. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Klio development. Any meta-KEP is also considered a Process KEP.
+
+
+Workflow
+--------
+
+The KEP process begins with a new idea for Klio.
+It is highly recommended that a single KEP contain a single key proposal or new idea.
+Small enhancements or patches often don't need a KEP and can be injected into the Klio development workflow with a pull request to the `Klio repo`_.
+The more focused the KEP, the more successful it tends to be. If in doubt, split your KEP into several well-focused ones.
+
+Each KEP must have a champion: someone who writes the KEP using the style and format described below, shepherds the discussions in the appropriate forums, and attempts to build consensus around the idea.
+The KEP champion (a.k.a. Author) should first attempt to ascertain whether the idea is suitable for a KEP.
+This may include posting to the #klio channel in the `Spotify FOSS Slack Organization`_.
+
+The proposal should be submitted as a draft KEP via a GitHub `pull request`_ to the ``docs/src/keps`` directory with the name ``kep-<n>.rst`` where ``<n>`` is an appropriately assigned three-digit number (e.g., ``kep-000.rst``).
+The draft must use the template defined in  :doc:`KEP X — Template and Instructions <kep-00X>`.
+
+At the earliest convenience, the PR should be merged regardless of whether it is accepted during discussion).
+Additional PRs may be made by the Author to update or expand the KEP, or by maintainers to set its status, discussion URL, etc.
+
+A Standards Track KEP consist of two parts: a design document and a reference implementation.
+It is generally recommended that at least a prototype implementation be co-developed with the KEP, as ideas that sound good in principle sometimes turn out to be impractical when subjected to the test of implementation.
+Often it makes sense for the prototype implementation to be made available as PR to the Klio repo (making sure to appropriately mark the PR as a ``WIP``).
+
+
+Review and Resolution
+^^^^^^^^^^^^^^^^^^^^^
+
+KEPs are discussed on their respective pull request. The possible paths of the status of KEPs are as follows:
+
+.. figure:: assets/kep_workflow.png
+    :alt: KEP Workflow
+
+    *KEP Workflow*
```

Comment:
For some reason, RTDs isn't updating - I use to have a graphviz diagram here (what RTDs shows now). But then I updated it to a simple PNG. Not sure why it's not being shown. But here's what it looks like:

<img width="812" alt="Screen Shot 2020-11-09 at 6 00 08 PM" src="https://user-images.githubusercontent.com/1191069/98606562-73abdc80-22b5-11eb-8c68-50c1ef60d55c.png">